lib/index.js

```diff
@@ -42,6 +47,11 @@ module.exports = {
    * @inner
    */
   helper: require('./helper'),
+  /**
+   * @type {typeof CodeceptJS.Helper}
```

Comment:
Does it duplicate?  Or maybe need remove helper on top?

Reply:
* `Helper` - a better naming, because it is class: `{ Helper } = require('codeceptjs')`
* `helper` for backwards compatibility

Ideally, we should move Helper into a standalone package...